### PR TITLE
add moisture and precipitation asserts to squall line CI

### PR DESCRIPTION
**Pull request by trontrytel**
### Description
This adds testing of different moisture and precipitation configurations to CI using squall line setup.

The tests are just a simple asserts at the end of a short run. Next up is adding squall line to QA.

- [x] Code follows the [style guidelines](https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/) OR N/A.
- [x] Unit tests are included OR N/A.
- [x] Code is exercised in an integration test OR N/A.
- [x] Documentation has been added/updated OR N/A.

### CODE REVIEW

File: experiments/AtmosLES/squall_line.jl [hunk 1]
```diff
@@ -196,7 +228,44 @@ function config_squall_line(FT, N, resolution, xmax, ymax, zmax, xmin, ymin)
     SHF = FT(10)
     ics = init_squall_line!
 
-    source = (Gravity(), rayleigh_sponge, CreateClouds()..., RainSnow_1M()...)
+    source = (Gravity(), rayleigh_sponge)
+    # RainSnow_1M()...)
```
Comment (charleskawczynski) [inline experiments/AtmosLES/squall_line.jl:232]:
```suggestion
```

**Review by charleskawczynski — APPROVED** [review]
LGTM

**Review by jakebolewski — APPROVED** [review]

**Comment by charleskawczynski** [comment]
Should we merge #1849 first?

**Comment by trontrytel** [comment]
> Should we merge #1849 first?

I don't think it matters which goes first. I only check the "max value greater than something" here. So I dont think the bug fix from #1849 will have an effect here.

**Pull request merged**